test/wallet_generator.js

```diff
@@ -18,36 +19,47 @@ describe('generating a ripple account', function() {
     wallet = new RippleWallet(HUMAN_SEED);
   });
 
+  it('should get the human readable seed', function() {
+    assert.strictEqual(wallet.secret, HUMAN_SEED);
+  });
+
   it('should get the master seed from the human readable seed', function() {
-    assert.strictEqual(wallet.getSeed().value, MASTER_SEED);
+    assert.strictEqual(wallet.getSeed().value.toUpperCase(), MASTER_SEED);
   });
 
-  it('should gen the correct address from a master seed in base 58 format', function() {
+  it('should generate the correct address from a master seed in base 58 format', function() {
     assert.strictEqual(wallet.getAddress().value, ACCOUNT_ID);
   });
 
-  it('should derive a private generator function from the master seed', function() {
+  it.skip('should derive a private generator function from the master seed', function() {
     assert.strictEqual(wallet.getPrivateGenerator().value, PRIVATE_GENERATOR);
   });
 
-  it('should derive a ecdsa private key with the private generator', function() {
+  it.skip('should derive a ecdsa private key with the private generator', function() {
     assert.strictEqual(wallet.getPrivateKey().value, PRIVATE_KEY);
   });
 
-  it('should derive a public generator elliptic curve point from the private key', function() {
+  it.skip('should derive a public generator elliptic curve point from the private key', function() {
     var publicGenerator = wallet.getPublicGenerator();
-    assert(publicGenerator instanceof PublicGenerator); 
+    assert(publicGenerator instanceof PublicGenerator);
     assert(publicGenerator.value instanceof sjcl.ecc.point);
     assert.strictEqual(publicGenerator.toString(), PUBLIC_GENERATOR);
   });
 
-  it('should generate the first public key from the public generator', function() {
-    assert.strictEqual(wallet.getPublicKey().value, PUBLIC_KEY);
+  it('should generate the account public key from a master seed', function() {
+    assert.strictEqual(wallet.getAccountPublicKey().value, ACCOUNT_PUBLIC_KEY);
   });
 
-  it('should convert the public key to a ripple account address', function() {
-    assert.strictEqual(wallet.getAccountId(), ACCOUNT_ID);
+  it('should generate the correct validation public key from a master seed', function() {
+    assert.strictEqual(wallet.getValidationPublicKey().value, VALIDATION_PUBLIC_KEY);
```

Comment:
It makes me sad that the pattern of having to call`.value` is adopted here again as I regret adding it in the first place. Can we please update the API to return only the actual value rather than an object `{ value: 'thekey' }`?